rest/model/distro.go

```diff
@@ -9,12 +9,24 @@ import (
 )
 
 // APIDistro is the model to be returned by the API whenever distros are fetched.
-// EVG-1717 will implement the remainder of the distro model.
 type APIDistro struct {
-	Name             APIString `json:"name"`
-	UserSpawnAllowed bool      `json:"user_spawn_allowed"`
-	Provider         APIString `json:"provider"`
-	ImageID          APIString `json:"image_id,omitempty"`
+	Name             APIString               `json:"name"`
+	UserSpawnAllowed bool                    `json:"user_spawn_allowed"`
+	Provider         APIString               `json:"provider"`
+	ProviderSettings *map[string]interface{} `json:"settings,omitempty"`
```

Comment:
There's no need for this to be a pointer in the API model. It's currently a pointer in the distro db model for a silly reason that I can't remember